### PR TITLE
replaced DAA root key by daaKeyId.  Added proper references

### PR DESCRIPTION
**Pull request by rlin1**
covers the DAA root key aspect as mentioned in #233.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1454,7 +1454,7 @@ authenticator over a set of contextual bindings. Each attestation statement form
     [=hash of the serialized client data=], and returns either:
     - An error indicating that the attestation is invalid, or
     - The attestation type, and the trust path of the attestation. This trust path is either empty (in case of
-        self-attestation), a DAA root key (in the case of Direct Anonymous Attestation), or a set of X.509 certificates.
+        self-attestation), an identifier of a [=DAA Issuer public key=] (in the case of Direct Anonymous Attestation), or a set of X.509 certificates.
```
Comment (equalsJeffH) [inline index.bs]: the string used in [[FIDOEcdaaAlgorithm]], for `[=DAA Issuer public key=]`, is "ECDAA-Issuer public key".  We should replicate that herein, yes?

Are all the references to "Direct Anonymous Attestation (DAA)" herein actually in regards to "elliptic curve DAA" ?  if so, that should be made clear, eg using "ECDAA" where we use "DAA" (regardless of case), yes?

**Comment by equalsJeffH** [comment]
also...

in [[!FIDOEcdaaAlgorithm]], these terms..

    DAA-Issuer
    DAA-Verify
    DAA-Sign

..are actually of the form "ECDAA-foo"

I suggest editing this parag from the {#attestation-formats} section..

>: <dfn>Direct Anonymous Attestation</dfn> (DAA)
:: In this case, the Authenticator receives DAA credentials from a single DAA-Issuer. These DAA credentials are used along with
    blinding to sign the [=attestation data=]. The concept of blinding avoids the DAA credentials being misused as global
    correlation handle. WebAuthn supports DAA using elliptic curve cryptography and bilinear pairings, called ECDAA (see
    [[FIDOEcdaaAlgorithm]]) in this specification.

..to be..

: <dfn>Direct Anonymous Attestation</dfn> (DAA)
:: In this case, an Authenticator receives DAA credentials from a given DAA-Issuer. These DAA credentials are used along with blinding to sign the [=attestation data=]. The concept of blinding avoids the DAA credentials being misused as a global correlation handle. WebAuthn employs the DAA variant based on elliptic curve cryptography and bilinear pairings, known as <dfn>ECDAA</dfn>, the details of which are specified in [[FIDOEcdaaAlgorithm]]. See the latter specification for the detailed definitions of <dfn>ECDAA-Issuer</dfn>, as well as the <dfn>ECDAA-Verify</dfn> and <dfn>ECDAA-Sign</dfn> operations. 

..and then have all subsequent occurrences of "DAA-foo" become "[=ECDAA-foo=]", as well as subsequent standalone occurrences of "DAA" become "[=ECDAA=]".

File: index.bs (hunk 1 above)
Comment (rlin1) [inline index.bs]: Yes.  Someone changed it from ECDAA to DAA some time ago.  Will revert back to ECDAA.

File: index.bs [hunk 2]
```diff
@@ -1778,8 +1778,10 @@ implementable by [=authenticators=] with limited resources (e.g., secure element
     :: The elements of this array contain the attestation certificate and its certificate chain, each encoded in X.509 format.
         The attestation certificate must be the first element in the array.
 
-    : daaKey
-    :: The DAA root key. The syntax for `eccPubKey` is defined in [[#sec-attestation-data]].
+    : daaKeyId
+    :: The <dfn>identifier of the DAA Issuer public key</dfn>.  This is the  
+        BigNumberToB encoding of the component "c" of the <dfn>DAA Issuer public key</dfn> 
+        as defined section 3.3, step 3.5 in [[!FIDOEcdaaAlgorithm]].
```
Comment (rlin1) [inline index.bs]: The challenge is that that c is a different object!

Comment (rlin1) [inline index.bs]: Step 3.5 in section 3.3. is the right one.

**Comment by rlin1** [comment]
Changes DAA to ECDAA since this is the only variant we are using.
@equalsJeffH: please verify that this addresses your concerns.

**Pull request merged**